src/__mocks__/globby.js

```diff
@@ -0,0 +1,10 @@
+const globby = jest.genMockFromModule('globby')
+
+globby.sync = args => {
+	if (!Array.isArray(args)) {
+		return [args]
+	}
+	return args
```

Comment:
I haven not mocked `globbing` here.

Ideally `/dir/*.json` should yield `['/dir/file1.json', 'dir/file2.json']`.
But are we to test how `globby` works here?